app/services/manifest_token_json_converter.rb

```diff
@@ -0,0 +1,22 @@
+class ManifestTokenJsonConverter
```

Comment:
ah ok.

for methods needed in multiple controllers, you can create a `Concern` in `app/controllers/concerns/`